### ISSUE TITLE
Object destructuring in @const

### ISSUE THREAD

**Issue by janosh** [sveltejs/eslint-plugin-svelte3#164]
False positive undefined variables when destructuring object keys in `@const` tag:

![Screen Shot 2022-01-30 at 20 23 58](https://user-images.githubusercontent.com/30958850/151716363-b38e88c3-fa60-4895-9fb7-3fcf776e914e.png)

[Minimal REPL](https://svelte.dev/repl/4bd682fa11194dcdbbf373d6d7b5d122?version=3.46.3)

```svelte
<script>
	const objs = [{a:1, b:2}, {a:3, b:4}]
</script>

{#each objs as obj}
	{@const {a, b} = obj}
	{a} {b}
{/each}
```
